core/chains/evm/logpoller/log_poller.go

```diff
@@ -34,8 +34,8 @@ type LogPoller interface {
 	services.ServiceCtx
 	Replay(ctx context.Context, fromBlock int64) error
 	ReplayAsync(fromBlock int64)
-	RegisterFilter(filter Filter) error
-	UnregisterFilter(name string, q pg.Queryer) error
+	RegisterFilter(filter Filter, qopts ...pg.QOpt) error
+	UnregisterFilter(name string, qopts ...pg.QOpt) error
```

Comment:
@reductionista WDYT? I thought that unifying it with other LogPoller's functions will reduce confusion